cmd/run.go

```diff
@@ -343,6 +366,18 @@ func run(cmd *cobra.Command, args []string) {
 		tee.AttachRunner(server.NewSnipRunner(influx.Run))
 	}
 
+	// MySQL client
+	if viper.GetString("mysql.host") != "" {
+		mysql := server.NewMySQLClient(
+			viper.GetString("mysql.host"),
+			viper.GetString("mysql.user"),
+			viper.GetString("mysql.password"),
+			viper.GetString("mysql.database"),
+		)
+
+		tee.AttachRunner(server.NewSnipRunner(mysql.Run))
+	}
```

Comment:
IMHO: What if I want to connect to pgsql or sqlite or mssql? At least provide an adapter/interface where other developers can integrate their desired DB. Also the CLI args should be then: `db.type=mysql`, `db.host`, `db.user` and so on